### ISSUE TITLE
Docker Compose project name not set

### ISSUE THREAD

**Issue by kc9jud** [bitwarden/self-host#14]
Docker Compose [defaults to the basename of the project directory](https://docs.docker.com/compose/reference/envvars/#compose_project_name) if no project name is set. Here, that means the project is simply called `docker`. This shows up in, for example, the output of `docker compose ls` and the name of the networks created. This should be set to `bitwarden` instead.

**Comment by joseph-flinn** [comment]
See comment on https://github.com/bitwarden/self-host/pull/15

**Comment by vipergts450** [comment]
I was also confounded by this when attempting a self-hosted install initially, but was able to work around it by creating a "docker-compose.override.yml" file in ./bwdata/docker which contains:

`version: '3'

networks:
  default:
    name: bitwarden_internal
  public:
    name: bitwarden_public
`

**Comment by vipergts450** [comment]
> I think this is still not fixed, despite the problems with #15. At minimum, Bitwarden should not be creating networks with names like `docker_default`.

I was also confounded by this when attempting a self-hosted install initially, but was able to work around it by creating a "docker-compose.override.yml" file in ./bwdata/docker which contains:

```
version: '3'

networks:
  default:
    name: bitwarden_internal
  public:
    name: bitwarden_public
```
